### PR TITLE
Add statistics for PMJ-based CSA and compute distance from C2-C3 and PMJ

### LINKED ISSUE #83: Compute distance between PMJ and C2-C3 disc
## Description
Compute distance from PMJ along extrapolated centerline from C2-C3 disc in processing pipeline.

### LINKED ISSUE #84: Add scatterplot of PMJ-based CSA vs C2-C3-based CSA in `compute_stats.py`
## TODO

- [x] Add PMJ-based CSA in `get_subject_info`
- [x] Add scatterplot of PMJ-based CSA vs C2-C3-based CSA in `compute_stats.py`
- [x] Add scatterplot of distance between C2-C3 disc and PMJ (related to #83 ) in `compute_stats.py`
- [x] Use PMJ-based CSA for statistics + normalization

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description
This PR addresses #83 and #84. 
Main changes include:
#### In `process_data.sh`:
* Compute distance from PMJ along extrapolated centerline from C2-C3 disc with the script `get_distance_pmj_disc.py`, save in a .csv file

To to this, I added the saving of the centerline coordinates since it is more precise than in the image of centerline_extrapolated --> https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/79752f834c9cb54b6a8b244068f1556a61dd8f75 

#### In `get_subject_info.py`:
* Read PMJ-based CSA and a column for it
* Read distance from PMJ and C2-C3 disc and add a column for it

#### In `compute_stats.py`:
* Compute mean, std, median, COV for both CSA measures
* Compute regression, normalization only with PMJ-based CSA
* Add scatter plot for C2-C3-based CSA vs PMJ-based CSA
* Add scatter plot for distance from PMJ and C2-C3 disc
image

## Linked issues
Resolves #84 and Resolves #83

### CODE REVIEW

**Comment by sandrinebedard** [comment]
I should maybe add in the `ylabel `of figure a) that CSA is measured at 60mm from PMJ

File: get_distance_pmj_disc.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Add padding to an image by mirring the image
```
Comment (jcohenadad) [inline get_distance_pmj_disc.py:3]:
```suggestion
# Add padding to an image by mirroring the image
```

**Review by jcohenadad — COMMENTED** [review]
This is absolutely fantastic! I **love** the scatterplot you generated, this will be extremely interesting for the scientific community. 

I just have a question: what is the distance from the PMJ and how did you choose it?

**Comment by sandrinebedard** [comment]
> I just have a question: what is the distance from the PMJ and how did you choose it?

Currently it is 60 mm!

Now that I have the distance between PMJ and C2-C3 disc, the mean is at 63 mm, but those results don't include yet the new R-L adjustment of PMJ label, so it should be arround that! I want to compute PMJ at the mean distance from C2-C3 disc

**Review by jcohenadad — APPROVED** [review]
amazing work!! 🤩  👏

**Pull request merged**